docs/fides/docs/installation/installation.md

```diff
@@ -0,0 +1,66 @@
+# Installation
+
+This page describes installation options that you might use when considering how to install fidesctl. Fidesctl consists of multiple components, possibly distributed among various physical or virtual machines. Therefore installation of fidesctl might be complex, depending on the options you choose.
```

Comment:
"Therefore installation of fidesctl might be complex, depending on the options you choose." seems a little ominous/discouraging. How about instead calling out that it's flexible and can meet the needs of complex environments?